ably/realtime_presence_test.go

```diff
@@ -96,3 +104,38 @@ func TestRealtimePresence_Sync250(t *testing.T) {
 		t.Fatalf("members3: %v", err)
 	}
 }
+
+var stateGuard = ^ably.StateEnum(0)
+
+var presTransitions = []ably.StateEnum{
```

Comment:
Shouldn't this be within `TestRealtimePresence_EnsureChannelIsAttached` since you are never going to use it outside of this test?